cl/recap/tasks.py

```diff
@@ -823,6 +825,88 @@ def process_recap_docket_history_report(self, pk):
     }
 
 
+@app.task(
+    bind=True, max_retries=3, interval_start=5 * 60, interval_step=5 * 60
+)
+def process_docket_iquery_page(self, pk):
```

Comment:
This all looks fine (aside from my one other comment), but did you look around to see if we had other code that'd do this instead? Alternatively, if we have other tasks that are very similar, can/should they be redone to use this code? I just fear repetition, though it's not the end of the world in this case.